### PR TITLE
feat: init add verifier name

### LINKED ISSUE #13211: Improve Verification data
A s a Improvement Idea for QR verification in Memberships, it would be interesting to  have the hability to add names when adding verifiers.

Later, when these verifiers  mark tickets as  checked in, it would show the adress and the specified name of the verifier. 

Name is important for when working with a lot of verifiers/partners with the possibility to add a name when adding the verifier  wallet address, so it appears:  
```
Checked In :  A few seconds ago   
Verifier:  Name (in case there is) / Wallet address (with ENS resolution)...
```

Captura de pantalla (580)

### PR DESCRIPTION
**Pull request by JoaoCampos89**
# Description

# Issues

Fixes #13211
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
@JoaoCampos89 Thanks for your contribution! Please open up _another_ PR to add your github login to the CLA bot config `.clabot`. Thanks!

**Comment by JoaoCampos89** [comment]
I signed this, but still gives the sign error. There is something that I need to do more?

**Comment by julien51** [comment]
>  I signed this, but still gives the sign error. There is something that I need to do more?

Let me check, but in the meantime, please fix the failing tests in Locksmith?

**Comment by JoaoCampos89** [comment]
@julien51 I needed to change locally the docker version, let me know if this change impacts the repo, I can revert it. Without that change I was not able to run docker

**Comment by JoaoCampos89** [comment]
@julien51 can you look why this is failing?

**Comment by julien51** [comment]
It looks like an HTTP connection isue to docker.io. Let me restrat the job

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: '3.2'
```
Comment (julien51) [inline docker/docker-compose.yml:1]: Why the change @JoaoCampos89 ? 
@clemsos do you think it is ok? We should do it in a separate PR!

File: locksmith/src/routes/v2/verifier.ts [hunk 2]
```diff
@@ -20,6 +20,12 @@ router.put(
   (req, res) => verifierController.addVerifier(req, res)
 )
 
+router.put(
```
Comment (julien51) [inline locksmith/src/routes/v2/verifier.ts:23]: Let's just reuse the previous route with an optional `verifierName` in the body.

File: packages/unlock-js/openapi.yml [hunk 3]
```diff
@@ -899,6 +911,34 @@ paths:
         500:
           $ref: '#/components/responses/500.InternalError'
 
+  /v2/api/verifier/{network}/{lockAddress}/{verifierAddress}/{verifierName}:
```
Comment (julien51) [inline packages/unlock-js/openapi.yml:914]: Same. Better to re-use the existing route as it is just a small addition,

File: unlock-app/src/components/interface/locks/Settings/forms/VerifierForm.tsx [hunk 4]
```diff
@@ -104,13 +112,30 @@ export const VerifierForm = ({
...
+  }) => {
     const resolvedAddress = await getAddressForName(address)
-    const response = await storage.createVerifier(
-      network,
-      lockAddress,
-      resolvedAddress
-    )
+    let response
+    if (name) {
+      response = await storage.createVerifierWithName(
```
Comment (julien51) [inline unlock-app/src/components/interface/locks/Settings/forms/VerifierForm.tsx:125]: If you use the same API call, then you can just make the `name` optional and then this is simpler/cleaner.

**Review by julien51 — CHANGES_REQUESTED** [review]
Great start @JoaoCampos89 !
I think we can simplify a small bit!

File: docker/docker-compose.yml (hunk 1 above)
Comment (JoaoCampos89) [reply docker/docker-compose.yml:1]: I needed to do this to run locally on my machine. I can revert this.

File: locksmith/src/routes/v2/verifier.ts (hunk 2 above)
Comment (JoaoCampos89) [reply locksmith/src/routes/v2/verifier.ts:23]: Ok, I will do asap

**Comment by JoaoCampos89** [comment]
@julien51 when you have time take a look. It was refactored as requested

File: docker/docker-compose.yml (hunk 1 above)
Comment (julien51) [inline docker/docker-compose.yml:1]: Is this change required? This should probably not be part of this PR.

**Review by julien51 — APPROVED** [review]

**Comment by julien51** [comment]
LGTM @JoaoCampos89 !! Thanks for your hard work on this!
